### ISSUE TITLE
flutter drive hangs after completion

### ISSUE THREAD

**Issue by matthew-carroll** [flutter/flutter#23615]
Executing `flutter drive ...` hangs after all tests are completed.  It takes a Ctrl+C to get control back.

I'm running the command on OSX against an iPhone 7 simulator.

**Comment by zoechi** [comment]
Is this related to #23614 ?

**Comment by matthew-carroll** [comment]
@zoechi No, I don't think so.  I'm guessing that #23614 is a bug in the code, whereas this issue is probably a bug in the tooling (though I'm not sure).
